x/profiles/types/codec.go

```diff
@@ -40,14 +40,14 @@ func RegisterInterfaces(registry types.InterfaceRegistry) {
 	registry.RegisterImplementations((*exported.VestingAccount)(nil), &Profile{})
 	registry.RegisterImplementations((*authtypes.GenesisAccount)(nil), &Profile{})
 	registry.RegisterInterface(
-		"desmos.profiles.v2.AddressData",
+		"desmos.profiles.v3.AddressData",
```

Comment:
Another question related to version. Should we set the version here to consensus version? e.g., `desmos.profiles.v7.AddressData`.

Reply:
As well as the [query.proto](https://github.com/desmos-labs/desmos/blob/leonardo/application-links-expiration-impl/proto/desmos/profiles/v3/query.proto#L23).

Reply:
No, these types should reflect the Protobuf versioning (which is different from the module consensus versioning). We should update them only when we change the Protobuf definitions, as they change less often than the module consensus version (see from v4 to v5 where we didn't change a single Protobuf version but we had to do a migration to add new keys)